### PR TITLE
Add ImmobilienScout24 as a service

### PR DESCRIPTION
**Pull request by ruempel**
ImmobilienScout24 supports 2FA via authenticator app, but does not provide a public documentation.

### CODE REVIEW

File: _data/other.yml [hunk 1]
```diff
@@ -203,6 +203,13 @@ websites:
       software: Yes
       doc: https://ifttt.com/settings#two-step-verification
 
+    - name: ImmobilienScout24
+      url: https://www.immobilienscout24.de
+      img: immobilienscout24.png
+      tfa: Yes
+      software: Yes
+      lang: de
```
Comment (stephengroat) [inline _data/other.yml]: could you remove the `lang` tag? we don't use it when `tfa: true`

**Review by stephengroat — CHANGES_REQUESTED** [review]
also, do you have any links to documentation that we could add to the `doc` tag?

**Comment by ruempel** [comment]
There is no publicly available documentation on 2FA. However, there are hints available at https://sso.immobilienscout24.de/account/verwaltung when logged in, but this link is useless for public audience.

**Review by stephengroat — APPROVED** [review]

**Comment by stephengroat** [comment]
@ruempel no problem. thanks for the addition and quick fixes!

**Review by fpigerre — APPROVED** [review]

**Pull request merged**